os/Kconfig

```diff
@@ -464,6 +464,10 @@ menu "Audio Support"
 source audio/Kconfig
 endmenu
 
+menu "Kernel Test Framework Support"
```

Comment:
1. Coudl you let me know the plan of this? I don't think this is important to be at first depth of menuconfig with current content. 
2. Without testcase example, this is not needed. Right? Then how about make it non-selectable config like ARCH_HAVE_XXX?

Reply:
I moved it under CONFIG_EXAMPLE_TESTCASES